packages/app/documents/__tests__/document.test.ts

```diff
@@ -960,6 +961,9 @@ describe('Documents', () => {
 
             expect(error).toBeNull()
             expect(document).not.toBeNull()
+
+            //* Publishing is a side effect; we do not await it in our code.
+            await wait(1000)
```

Comment:
Removing the `await`s in `packages/app/documents/service.ts` means that awaiting `changeDocumentState` no longer guarantees that the document queue has been written to.